cmd/release-controller/main.go

```diff
@@ -307,14 +325,14 @@ func (o *options) Run() error {
 	if err != nil {
 		return fmt.Errorf("unable to create image client: %v", err)
 	}
-	klog.V(4).Infof("1: %v", time.Now().Sub(start))
+	klog.V(4).Infof("1: %v", time.Since(start))
```

Comment:
All these lines can actually be removed now.

Reply:
Remove all `klog.V(4).Infof("1: %v", time.Since(start))` ?